ginga/canvas/types/astro.py

```diff
@@ -741,6 +743,211 @@ def move_to_pt(self, dst_pt):
         self.set_data_points([dst_pt])
 
 
+class Annulus2R(AnnulusMixin, OnePointTwoRadiusMixin, CompoundObject):
+    """
+    Special compound object to handle annulus shape that
+    consists of two objects, (one center point, plus two radii).
```

Comment:
This explanation wasn't that clear to me. I thought you mean two different annuli bound by the same center point. But after playing with it in `Drawing`, this is basically one annulus but with different values along X and Y.

Reply:
Right. Our existing `Annulus` type only allows for objects with a single radius.  But `regions` has annulus shapes that have 2 radii; i.e. the width can be longer than the height, etc.